includes/admin/customers/customers.php

```diff
@@ -709,39 +712,42 @@ function edd_customers_view( $customer = null ) {
 								: '';
 							?>
 						</td>
-						<td>
+						<td data-colname="<?php esc_attr_e( 'City', 'easy-digital-downloads' ); ?>">
 							<?php
 							echo ! empty( $address->city )
-								? esc_html( $address->city )
+								? esc_attr( $address->city )
```

Comment:
This should be able to stay `esc_html()`, I think? The other columns have changed as well except for Postal Code.